apps/api/src/app/subscribers/dtos/get-in-app-notification-feed-for-subscriber.dto.ts

```diff
@@ -26,4 +26,12 @@ export class GetInAppNotificationsFeedForSubscriberDto extends PaginationRequest
 
   @ApiPropertyOptional({ required: false, type: Boolean })
   seen: boolean;
+
+  @ApiPropertyOptional({
+    required: false,
+    type: 'string',
+    description: 'Base64 encoded string of the partial payload JSON object',
+    example: 'btoa(JSON.stringify({ foo: 123 })) results in base64 encoded string like eyJmb28iOjEyM30=',
+  })
+  payload?: string;
```

Comment:
The two feed endpoints now do take the payload query param which is the base64 encoded string of the partial payload JSON object. Check the description for more info.